*** tests/A-13-03-M-ok.out	2022-10-30 22:16:20.721850658 +0000
--- tests/A-13-03-M-ok.outhyp	2022-11-01 15:22:26.467825584 +0000
***************
*** 34,41 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: VIDEO|1|969015|969014|0|0|ONGOING
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 34,40 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
